lib/v2/sec-in/radar.js

```diff
@@ -0,0 +1,11 @@
+module.exports = {
+    'sec-in.com': {
+        _name: 'SecIN信息安全技术社区',
+        '.': [
+            {
+                title: 'SecIN信息安全技术社区',
```

Comment:
https://github.com/DIYgod/RSSHub/blob/07d6aaa1e5dcd78e68d00ee3fb20ea196d098b88/docs/bbs.md?plain=1#L355
```suggestion
                title: '最新文章',
```